backend/routes/passwordRoute.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const router = express.Router();
+const { PrismaClient } = require("@prisma/client");
+const prisma = new PrismaClient();
+
+router.post('/check-password', async(req, res) => {
```

Comment:
The handler function and its logic that you have currently living in the endpoints should be extracted into functions and then just import it in the routes file accordingly. The functions should live in a folder called handlers. So in your case you'd import a method that handles the resetpassword logic from handlers/resetPassword.js or something along those lines.

Reply:
i believe i did this correctly, i made another folder called Handlers and transferred the function logic into that file. then just kept the route in the "PasswordRoute.js" folder.